genesis-info/tn.log

```diff
@@ -0,0 +1,358 @@
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1018k    0 1018k    0     0  2555k      0 --:--:-- --:--:-- --:--:-- 2558k
+
+fetching path ‘/nix/store/yh911hbispm1yrwg6h8bvlivs5xv2z02-snap-server-1.0.2.0’...
+
+*** Downloading ‘https://hydra.iohk.io/nar/yh911hbispm1yrwg6h8bvlivs5xv2z02-snap-server-1.0.2.0’ (signed by ‘hydra.iohk.io’) to ‘/nix/store/yh911hbispm1yrwg6h8bvlivs5xv2z02-snap-server-1.0.2.0’...
+
100 44244    0 44244    0     0   191k      0 --:--:-- --:--:-- --:--:--  192k
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
+
100  517k    0  517k    0     0  1718k      0 --:--:-- --:--:-- --:--:-- 1723k
+
+fetching path ‘/nix/store/pl7f3v357yav0i1x69f2401d4w113s6q-ekg-0.4.0.13’...
+
+*** Downloading ‘https://hydra.iohk.io/nar/pl7f3v357yav0i1x69f2401d4w113s6q-ekg-0.4.0.13’ (signed by ‘hydra.iohk.io’) to ‘/nix/store/pl7f3v357yav0i1x69f2401d4w113s6q-ekg-0.4.0.13’...
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  112k    0  112k    0     0   465k      0 --:--:-- --:--:-- --:--:--  466k
+
+fetching path ‘/nix/store/1yhsgimnbfiyqssgmcfcg67s8if8hvg0-cardano-sl-0.4.4’...
+
+*** Downloading ‘https://hydra.iohk.io/nar/1yhsgimnbfiyqssgmcfcg67s8if8hvg0-cardano-sl-0.4.4’ (signed by ‘hydra.iohk.io’) to ‘/nix/store/1yhsgimnbfiyqssgmcfcg67s8if8hvg0-cardano-sl-0.4.4’...
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2655k    0 2655k    0     0  4022k      0 --:--:-- --:--:-- --:--:-- 4023k
100 9150k    0 9150k    0     0  5508k      0 --:--:--  0:00:01 --:--:-- 5508k
100 14.8M    0 14.8M    0     0  5712k      0 --:--:--  0:00:02 --:--:-- 5711k
100 20.8M    0 20.8M    0     0  5822k      0 --:--:--  0:00:03 --:--:-- 5822k
100 22.0M    0 22.0M    0     0  5827k      0 --:--:--  0:00:03 --:--:-- 5826k
+
+Running command: /nix/store/1yhsgimnbfiyqssgmcfcg67s8if8hvg0-cardano-sl-0.4.4/bin/cardano-keygen -f /home/ielectric/dev/serokell/daedalus/cardano-sl/scripts/generate/../../secrets/secret-{}.key -m 14 -n 12000 --richmen-share 0.94 --testnet-stake 19072918462000000 --utxo-file /home/ielectric/dev/serokell/daedalus/cardano-sl/scripts/generate/../avvm-files/utxo-dump-last-new.json --randcerts --blacklisted /home/ielectric/dev/serokell/daedalus/cardano-sl/scripts/generate/../avvm-files/full_blacklist.js --fake-avvm-seed-pattern avvm/fake-{}.seed --fake-avvm-entries 100
```

Comment:
It's being logged automatically, we didn't want to write complex scripts for handling that (i.e. that will log accurately). This file is for double-checking mostly